hr_employee_calendar_planning/models/hr_employee.py

```diff
@@ -48,6 +48,21 @@ class HrEmployee(models.Model):
         copy=True,
     )
 
+    @api.model
+    def default_get(self, fields):
+        """Set calendar_ids default value to cover all use cases."""
+        vals = super().default_get(fields)
+        if (
+            not config["test_enable"]
```

Comment:
Why do you need this condition?

Reply:
The change is necessary to avoid errors in the tests of other modules (https://github.com/OCA/hr/actions/runs/5441518693/jobs/9895568077#step:8:379).

Reply:
But this is because there's a missing `ondelete="cascade"` here:

https://github.com/OCA/hr/blob/6a0b62b59bc349c85a06faadc70edfc4976d718e/hr_employee_calendar_planning/models/hr_employee.py#L163

and `ondelete="restrict"` here:

https://github.com/OCA/hr/blob/6a0b62b59bc349c85a06faadc70edfc4976d718e/hr_employee_calendar_planning/models/hr_employee.py#L166